src/pages/ViolationStudent.jsx

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
why comment an entire file?

Reply:
sorry sir i commented out the code in Violation Student and Violation Guest because those are different tickets